### PR TITLE
Get version for tags across all branches, not only the current branch.

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by notlee** [comment]
I think we actually don't want to do this. If a beta is released on one branch we still want to be able to release from the main branch

File: index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ function getLabelNamesFromPullRequest(payload) {
 
 async function getLatestTagOnCurrentBranch() {
   const { stdout: tags } = await exec(
-    'git describe --abbrev=0 --tags || echo 0'
+    'git describe --tags `git rev-list --tags --max-count=1` || echo 0'
```
Comment (JakeChampion) [inline index.js:20]: Why is this change required? If it is required, we should likely change the name of the function as it is no longer getting the tags for the current branch

Comment (notlee) [reply index.js:20]: >I think we actually don't want to do this. If a beta is released on one branch we still want to be able to release from the main branch

I made this change because it looks like, for some reason, the latest tags aren't on the master branch. Release logs:
```
 Found labels: "release:patch".
Calculating the version for a "patch" release.
Found "v1.1.2" as the latest release.
Creating a "v1.1.3" release.
Error: Command failed: git tag v1.1.3
fatal: tag 'v1.1.3' already exists
```

Comment (notlee) [reply index.js:20]: I think it's because we commit and tag the build now

Comment (JakeChampion) [reply index.js:20]: The newly created commit, is that ever pushed onto the default branch? (`master` right now is our default branch)
